Gopkg.toml

```diff
@@ -27,19 +27,19 @@ required = [
 
 [[constraint]]
   name = "github.com/container-storage-interface/spec"
-  version = "v0.2.0"
+  version = "v0.3.0"
 
-[[constraint]]
+[[override]]
```

Comment:
unrelated to the goal of this PR: unclear why gogo is listed as a dependency here. we should remove this.

Reply:
This (should?) already be removed through rebase.